app/pages/amplience/page/index.jsx

```diff
@@ -0,0 +1,143 @@
+ * The page is created for demonstration purposes.
+ * The page renders SEO metadata and a few promotion
+ * categories and products, data is from local file.
+ */
+const ContentPage = ({page}) => {
+    //const intl = useIntl()
+
+    const RTV = useContext(RealtimeVisualization)
+    let removeChangedSubscription = undefined
+    const [pageModel, setPageModel] = useState(undefined)
+
+    useEffect(() => {
+        setPageModel(page)
+        console.log('page', pageModel)
+    }, [page])
+
+    useEffect(() => {
+        if (RTV.ampVizSdk !== null) {
+            RTV.ampVizSdk.form.saved(() => {
+                window.location.reload()
+            })
+
+            removeChangedSubscription = RTV.ampVizSdk.form.changed((model) => {
+                // handle form model change
+                setPageModel(model.content)
+            })
+        }
+
+        return () => {
+            if (removeChangedSubscription != undefined) {
+                removeChangedSubscription()
+            }
+        }
+    }, [RTV.ampVizSdk])
+
+    return (
+        <Box data-testid="amplience-page" layerStyle="page">
+            {pageModel == undefined ? (
+                <Skeleton height="20px" />
+            ) : (
+                <>
+                    <Seo
+                        title={pageModel.seo.title}
+                        description={pageModel.seo.description}
+                        keywords={pageModel.seo.keywords}
+                        noIndex={pageModel.seo.noindex}
+                    />
+                    <Heading
+                        as="h1"
+                        fontSize={{base: '4xl', md: '5xl', lg: '6xl'}}
+                        maxWidth={{base: '75%', md: '50%', lg: 'md'}}
+                    >
+                        {pageModel.seo.title}
+                    </Heading>
+                </>
+            )}
+            {pageModel == undefined ? (
+                <Skeleton height="200px" />
+            ) : (
+                <>
+                    {pageModel.content.map((item) => {
+                        return <AmplienceWrapper key={item._meta.deliveryId} content={item} />
+                    })}
+                </>
+            )}
+        </Box>
+    )
+}
+
+ContentPage.getTemplateName = () => 'contentpage'
+
+ContentPage.shouldGetProps = ({previousLocation, location}) =>
+    !previousLocation || previousLocation.pathname !== location.pathname
+
+ContentPage.getProps = async ({res, params, location, api, ampClient}) => {
+    const {pageId} = params
+
+    if (res) {
+        res.set('Cache-Control', `max-age=${MAX_CACHE_AGE}`)
+    }
+
+    const site = resolveSiteFromUrl(location.pathname)
+    const l10nConfig = site.l10n
+    const targetLocale = getTargetLocale({
+        getUserPreferredLocales: () => {
+            const {locale} = api.getConfig()
+            return [locale]
+        },
+        l10nConfig
+    })
+
+    var page
+
+    console.log('params: ', params)
```

Comment:
Remove console logs, now that work is complete?

Reply:
updated